components/samsung_ac/protocol.h

```diff
@@ -84,6 +85,7 @@ namespace esphome
             optional<bool> power;
             optional<Mode> mode;
             optional<float> target_temp;
+            optional<float> target_water_temp;
```

Comment:
You added the water temperature to the request, but you did not extended the code witch sents the request.

Please also add a not supported message to non_nasa code.

Reply:
😬 I'm not sure where to add the "not supported message to non_nasa code" -- can you link to an example ?
same for water temp